tuf/api/metadata.py

```diff
@@ -61,14 +61,19 @@
     SignedSerializer,
 )
 
+_ROOT: str = "root"
+_SNAPSHOT: str = "snapshot"
+_TARGETS: str = "targets"
+_TIMESTAMP: str = "timestamp"
```

Comment:
I can see why you added these, re-ordering the code wouldn't be enough to avoid them...

I might have just hard coded the strings in the three places (to make the code in those places easier to read) but I guess this works, it's just one more indirection.

That said, these variables should not need type annotations, right? I'd like to not include annotations unless they are necessary